src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -851,38 +405,17 @@ FixedwingPositionControl::get_demanded_airspeed()
 float
 FixedwingPositionControl::calculate_target_airspeed(float airspeed_demand)
 {
-	float airspeed;
-
-	if (_airspeed_valid) {
```

Comment:
If you look at the old code it's only used in this function to calculate airspeed_error, which is never used. One of the reasons I started doing this is the controller had a lot of code that looked real, but was actually doing nothing. I've previously done several passes of deleting.